examples/ex1_plot_map.py

```diff
@@ -0,0 +1,45 @@
+"""
+=======================
+Three dimensional plots
+=======================
+sunpy can interface with the `pyvista` package to produce interactive 3D plots.
+"""
+###############################################################################
+# Start by importing the required modules
+from sys import platform
+
+import pyvista as pv
+
+import astropy.constants as const
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from sunpy.data.sample import AIA_193_IMAGE
+from sunpy.map import Map
+
+from sunkit_pyvista import SunpyPlotter
+
+if platform == "linux":
+    pv.start_xvfb()
```

Comment:
How does pyvista handle the other platforms?

Reply:
So when running on a vm pyvista warns the user that `PyVista will likely segfault when rendering`. Turns out this check is just disabled for windows because of it taking too long.
This doesn't seem to affect the tox virtual env so I'll remove it for now.